### PR TITLE
[Merged by Bors] - fix: add missing `_root_`

### PR DESCRIPTION
**Pull request by eric-wieser**
Mathport doesn't understand this, and apparently nor do many of the humans fixing the errors it creates.

If your `#align` statement complains the def doesn't exist, **don't change the #align**; work out why it doesn't exist instead.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Hom/Equiv/Units/GroupWithZero.lean [hunk 1]
```diff
@@ -33,9 +33,9 @@ protected def mulLeft₀ (a : G) (ha : a ≠ 0) : Perm G :=
 #align equiv.mul_left₀_symm_apply Equiv.mulLeft₀_symm_apply
 #align equiv.mul_left₀_apply Equiv.mulLeft₀_apply
 
-theorem mulLeft_bijective₀ (a : G) (ha : a ≠ 0) : Function.Bijective ((· * ·) a : G → G) :=
+theorem _root_.mulLeft_bijective₀ (a : G) (ha : a ≠ 0) : Function.Bijective ((· * ·) a : G → G) :=
```
Comment (Ruben-VandeVelde) [inline Mathlib/Algebra/Hom/Equiv/Units/GroupWithZero.lean:36]: Should this also be `mul_left`? But maybe it's better to keep this pr focused tightly.

**Comment by eric-wieser** [comment]
@Parcly-Taxel, your latest commit is great, but I don't think it belongs in this PR. Please revert it and make a new PR with that change.

**Comment by Parcly-Taxel** [comment]
> @Parcly-Taxel, your latest commit is great, but I don't think it belongs in this PR. Please revert it and make a new PR with that change.

I needed to do it to get the full list of possible misalignments:

```python
#!/usr/bin/env python3

with open("matches", 'r') as f:
    for l in f:
        fn, _, ln = l.strip().partition(":")
        try:
            name3, name4 = ln.split(" ")[1:3]
            if name3.count(".") < name4.count("."):
                print(fn, "->", name3, "/", name4)
        except ValueError:
            print("*****", l)
```

```
Mathlib/Algebra/GroupWithZero/Defs.lean -> zero_mul / MulZeroClass.zero_mul
Mathlib/Algebra/GroupWithZero/Defs.lean -> mul_zero / MulZeroClass.mul_zero
Mathlib/Algebra/Module/Pi.lean -> function.module / Pi.Function.module
Mathlib/Algebra/Module/Projective.lean -> module.projective_of_basis / Module.Projective.of_basis
Mathlib/Algebra/Module/Projective.lean -> module.projective_of_free / Module.Projective.of_free
Mathlib/Algebra/Module/Projective.lean -> module.projective_of_lifting_property' / Module.Projective.of_lifting_property'
Mathlib/Algebra/Module/Projective.lean -> module.projective_of_lifting_property / Module.Projective.of_lifting_property
Mathlib/Algebra/Order/Monoid/OrderDual.lean -> ordered_cancel_add_comm_monoid.to_contravariant_class / OrderDual.OrderedCancelAddCommMonoid.to_contravariantClass
Mathlib/Analysis/Normed/Order/Lattice.lean -> normed_lattice_add_comm_group_to_ordered_add_comm_group / NormedLatticeAddCommGroup.toOrderedAddCommGroup
Mathlib/Analysis/Normed/Order/Lattice.lean -> normed_lattice_add_comm_group_has_continuous_inf / NormedLatticeAddCommGroup.continuousInf
Mathlib/Analysis/Normed/Order/Lattice.lean -> normed_lattice_add_comm_group_has_continuous_sup / NormedLatticeAddCommGroup.continuousSup
Mathlib/Analysis/Normed/Order/Lattice.lean -> normed_lattice_add_comm_group_topological_lattice / NormedLatticeAddCommGroup.toTopologicalLattice
Mathlib/CategoryTheory/Groupoid/VertexGroup.lean -> category_theory.functor.map_vertex_group / CategoryTheory.Groupoid.CategoryTheory.Functor.mapVertexGroup
Mathlib/CategoryTheory/Limits/Shapes/Biproducts.lean -> category_theory.limits.bicone_X / CategoryTheory.Limits.Bicone.pt
Mathlib/Data/Complex/Basic.lean -> set.re_prod_im / Complex.Set.reProdIm
Mathlib/Data/DList/Basic.lean -> dlist.join / Std.DList.join
Mathlib/Data/DList/Basic.lean -> dlist.lazy_of_list / Std.DList.lazy_ofList
Mathlib/Data/DList/Basic.lean -> dlist_singleton / Std.DList_singleton
Mathlib/Data/DList/Basic.lean -> dlist_lazy / Std.DList_lazy
Mathlib/Data/DList/Basic.lean -> dlist.to_list_of_list / Std.DList.toList_ofList
Mathlib/Data/DList/Basic.lean -> dlist.of_list_to_list / Std.DList.ofList_toList
Mathlib/Data/DList/Instances.lean -> dlist.list_equiv_dlist / Std.DList.listEquivDList
Mathlib/Data/Finset/Slice.lean -> set.sized.card_le / Finset.Set.Sized.card_le
Mathlib/Data/List/Range.lean -> list.range_core / List.range.loop
Mathlib/Data/Nat/Prime.lean -> irreducible_iff_nat_prime / Nat.irreducible_iff_nat_prime
Mathlib/Data/PFun.lean -> part.bind_comp / PFun.Part.bind_comp
Mathlib/Data/Real/CauSeq.lean -> rat_sup_continuous_lemma / CauSeq.rat_sup_continuous_lemma
Mathlib/Data/Real/CauSeq.lean -> rat_inf_continuous_lemma / CauSeq.rat_inf_continuous_lemma
Mathlib/Data/Set/Finite.lean -> supr_infi_of_monotone / Set.supᵢ_infᵢ_of_monotone
Mathlib/Data/Set/Finite.lean -> supr_infi_of_antitone / Set.supᵢ_infᵢ_of_antitone
Mathlib/Data/Set/Finite.lean -> infi_supr_of_monotone / Set.infᵢ_supᵢ_of_monotone
Mathlib/Data/Set/Finite.lean -> infi_supr_of_antitone / Set.infᵢ_supᵢ_of_antitone
Mathlib/Data/Sign.lean -> sign / SignType.sign
Mathlib/GroupTheory/DoubleCoset.lean -> doset / Doset.doset
Mathlib/Init/Align.lean -> punit_eq / PUnit.subsingleton
Mathlib/Init/Align.lean -> punit_eq_star / PUnit.eq_punit
Mathlib/Init/Core.lean -> gt / GT.gt
Mathlib/Init/Core.lean -> ge / GE.ge
Mathlib/Init/Core.lean -> sizeof / SizeOf.sizeOf
Mathlib/Init/Core.lean -> nat_add_zero / Nat.add_zero
Mathlib/Init/Data/Bool/Lemmas.lean -> band_self / Bool.and_self
Mathlib/Init/Data/Bool/Lemmas.lean -> band_tt / Bool.and_true
Mathlib/Init/Data/Bool/Lemmas.lean -> band_ff / Bool.and_false
Mathlib/Init/Data/Bool/Lemmas.lean -> tt_band / Bool.true_and
Mathlib/Init/Data/Bool/Lemmas.lean -> ff_band / Bool.false_and
Mathlib/Init/Data/Bool/Lemmas.lean -> bor_self / Bool.or_self
Mathlib/Init/Data/Bool/Lemmas.lean -> bor_tt / Bool.or_true
Mathlib/Init/Data/Bool/Lemmas.lean -> bor_ff / Bool.or_false
Mathlib/Init/Data/Bool/Lemmas.lean -> tt_bor / Bool.true_or
Mathlib/Init/Data/Bool/Lemmas.lean -> ff_bor / Bool.false_or
Mathlib/Init/Data/Bool/Lemmas.lean -> bnot_bnot / Bool.not_not
Mathlib/Init/Data/Bool/Lemmas.lean -> cond_a_a / Bool.cond_self
Mathlib/Init/Data/Bool/Lemmas.lean -> bxor_self / Bool.xor_self
Mathlib/Init/Data/Bool/Lemmas.lean -> bxor_tt / Bool.xor_true
Mathlib/Init/Data/Bool/Lemmas.lean -> bxor_ff / Bool.xor_false
Mathlib/Init/Data/Bool/Lemmas.lean -> tt_bxor / Bool.true_xor
Mathlib/Init/Data/Bool/Lemmas.lean -> ff_bxor / Bool.false_xor
Mathlib/Init/Data/Bool/Lemmas.lean -> tt_eq_ff_eq_false / Bool.true_eq_false_eq_False
Mathlib/Init/Data/Bool/Lemmas.lean -> ff_eq_tt_eq_false / Bool.false_eq_true_eq_False
Mathlib/Init/Data/Bool/Lemmas.lean -> eq_ff_eq_not_eq_tt / Bool.eq_false_eq_not_eq_true
Mathlib/Init/Data/Bool/Lemmas.lean -> eq_tt_eq_not_eq_ft / Bool.eq_true_eq_not_eq_false
Mathlib/Init/Data/Bool/Lemmas.lean -> eq_ff_of_not_eq_tt / Bool.eq_false_of_not_eq_true
Mathlib/Init/Data/Bool/Lemmas.lean -> eq_tt_of_not_eq_ff / Bool.eq_true_of_not_eq_false
Mathlib/Init/Data/Bool/Lemmas.lean -> band_eq_true_eq_eq_tt_and_eq_tt / Bool.and_eq_true_eq_eq_true_and_eq_true
Mathlib/Init/Data/Bool/Lemmas.lean -> bor_eq_true_eq_eq_tt_or_eq_tt / Bool.or_eq_true_eq_eq_true_or_eq_true
Mathlib/Init/Data/Bool/Lemmas.lean -> bnot_eq_true_eq_eq_ff / Bool.not_eq_true_eq_eq_false
Mathlib/Init/Data/Bool/Lemmas.lean -> band_eq_false_eq_eq_ff_or_eq_ff / Bool.and_eq_false_eq_eq_false_or_eq_false
Mathlib/Init/Data/Bool/Lemmas.lean -> bor_eq_false_eq_eq_ff_and_eq_ff / Bool.or_eq_false_eq_eq_false_and_eq_false
Mathlib/Init/Data/Bool/Lemmas.lean -> bnot_eq_ff_eq_eq_tt / Bool.not_eq_false_eq_eq_true
Mathlib/Init/Data/Bool/Lemmas.lean -> coe_ff / Bool.coe_false
Mathlib/Init/Data/Bool/Lemmas.lean -> coe_tt / Bool.coe_true
Mathlib/Init/Data/Bool/Lemmas.lean -> coe_sort_ff / Bool.coe_sort_false
Mathlib/Init/Data/Bool/Lemmas.lean -> coe_sort_tt / Bool.coe_sort_true
Mathlib/Init/Data/Bool/Lemmas.lean -> to_bool_iff / Bool.decide_iff
Mathlib/Init/Data/Bool/Lemmas.lean -> to_bool_true / Bool.decide_true
Mathlib/Init/Data/Bool/Lemmas.lean -> to_bool_tt / Bool.decide_true
Mathlib/Init/Data/Bool/Lemmas.lean -> of_to_bool_true / Bool.of_decide_true
Mathlib/Init/Data/Bool/Lemmas.lean -> bool_iff_false / Bool.bool_iff_false
Mathlib/Init/Data/Bool/Lemmas.lean -> bool_eq_false / Bool.bool_eq_false
Mathlib/Init/Data/Bool/Lemmas.lean -> to_bool_ff_iff / Bool.decide_false_iff
Mathlib/Init/Data/Bool/Lemmas.lean -> to_bool_ff / Bool.decide_false
Mathlib/Init/Data/Bool/Lemmas.lean -> of_to_bool_ff / Bool.of_decide_false
Mathlib/Init/Data/Bool/Lemmas.lean -> to_bool_congr / Bool.decide_congr
Mathlib/Init/Data/Bool/Lemmas.lean -> bor_coe_iff / Bool.or_coe_iff
Mathlib/Init/Data/Bool/Lemmas.lean -> band_coe_iff / Bool.and_coe_iff
Mathlib/Init/Data/Bool/Lemmas.lean -> bxor_coe_iff / Bool.xor_coe_iff
Mathlib/Init/Data/Bool/Lemmas.lean -> ite_eq_tt_distrib / Bool.ite_eq_true_distrib
Mathlib/Init/Data/Bool/Lemmas.lean -> ite_eq_ff_distrib / Bool.ite_eq_false_distrib
Mathlib/Init/Data/Int/Basic.lean -> int.coe_nat_inj / Int.ofNat.inj
Mathlib/Init/Data/Int/Basic.lean -> int.add_assoc_aux1 / Int.add_assoc.aux1
Mathlib/Init/Data/Int/Basic.lean -> int.add_assoc_aux2 / Int.add_assoc.aux2
Mathlib/Init/Logic.lean -> exists_of_exists_unique / ExistsUnique.exists
Mathlib/Init/Logic.lean -> unique_of_exists_unique / ExistsUnique.unique
Mathlib/Init/Logic.lean -> exists_imp_exists / Exists.imp
Mathlib/Init/Logic.lean -> arbitrary / Inhabited.default
Mathlib/Init/Logic.lean -> of_as_true / AsTrue.get
Mathlib/Init/Logic.lean -> mk_equivalence / Equivalence.mk
Mathlib/LinearAlgebra/Basis.lean -> eq_bot_of_rank_eq_zero / Basis.eq_bot_of_rank_eq_zero
Mathlib/LinearAlgebra/InvariantBasisNumber.lean -> noetherian_ring_strong_rank_condition / IsNoetherianRing.strongRankCondition
Mathlib/LinearAlgebra/Ray.lean -> same_ray_comm / SameRay.sameRay_comm
Mathlib/LinearAlgebra/Ray.lean -> same_ray_nonneg_smul_right / SameRay.sameRay_nonneg_smul_right
Mathlib/LinearAlgebra/Ray.lean -> same_ray_pos_smul_right / SameRay.sameRay_pos_smul_right
Mathlib/LinearAlgebra/Ray.lean -> same_ray_nonneg_smul_left / SameRay.sameRay_nonneg_smul_left
Mathlib/LinearAlgebra/Ray.lean -> same_ray_pos_smul_left / SameRay.sameRay_pos_smul_left
Mathlib/LinearAlgebra/Ray.lean -> same_ray_map_iff / SameRay.sameRay_map_iff
Mathlib/Logic/Basic.lean -> not_not / Classical.not_not
Mathlib/Logic/Basic.lean -> exists_imp_exists' / Exists.imp'
Mathlib/Order/Basic.lean -> ge.le / GE.ge.le
Mathlib/Order/Basic.lean -> gt.lt / GT.gt.lt
Mathlib/RingTheory/Polynomial/Eisenstein/Basic.lean -> polynomial.monic.leading_coeff_not_mem / Polynomial.IsEisensteinAt.Polynomial.Monic.leadingCoeff_not_mem
Mathlib/RingTheory/Polynomial/Eisenstein/Basic.lean -> polynomial.monic.is_eisenstein_at_of_mem_of_not_mem / Polynomial.IsEisensteinAt.Polynomial.Monic.isEisensteinAt_of_mem_of_not_mem
Mathlib/RingTheory/SimpleModule.lean -> jordan_holder_module / JordanHolderModule.instJordanHolderLattice
Mathlib/Tactic/Group.lean -> tactic.group.zpow_trick / Mathlib.Tactic.Group.zpow_trick
Mathlib/Tactic/Group.lean -> tactic.group.zsmul_trick / Mathlib.Tactic.Group.zsmul_trick
Mathlib/Tactic/Group.lean -> tactic.group.zpow_trick_one / Mathlib.Tactic.Group.zpow_trick_one
Mathlib/Tactic/Group.lean -> tactic.group.zsmul_trick_zero / Mathlib.Tactic.Group.zsmul_trick_zero
Mathlib/Tactic/Group.lean -> tactic.group.zpow_trick_one' / Mathlib.Tactic.Group.zpow_trick_one'
Mathlib/Tactic/Group.lean -> tactic.group.zsmul_trick_zero' / Mathlib.Tactic.Group.zsmul_trick_zero'
Mathlib/Tactic/ToLevel.lean -> reflected_univ / Lean.ToLevel
Mathlib/Tactic/ToLevel.lean -> reflected_univ.lvl / Lean.ToLevel.toLevel
Mathlib/Topology/Algebra/InfiniteSum/Order.lean -> finite_of_summable_const / Set.Finite.of_summable_const
Mathlib/Topology/MetricSpace/Basic.lean -> uniform_space_of_dist / UniformSpace.ofDist
Mathlib/Topology/MetricSpace/Polish.lean -> polish_space.complete_copy / TopologicalSpace.Opens.CompleteCopyₓ
Mathlib/Topology/MetricSpace/Polish.lean -> polish_space.has_dist_complete_copy / TopologicalSpace.Opens.CompleteCopy.instDistₓ
Mathlib/Topology/MetricSpace/Polish.lean -> polish_space.dist_complete_copy_eq / TopologicalSpace.Opens.CompleteCopy.dist_eqₓ
Mathlib/Topology/MetricSpace/Polish.lean -> polish_space.dist_le_dist_complete_copy / TopologicalSpace.Opens.CompleteCopy.dist_val_le_distₓ
Mathlib/Topology/MetricSpace/Polish.lean -> polish_space.complete_copy_metric_space / TopologicalSpace.Opens.CompleteCopy.instMetricSpaceₓ
Mathlib/Topology/MetricSpace/Polish.lean -> polish_space.complete_space_complete_copy / TopologicalSpace.Opens.CompleteCopy.instCompleteSpaceₓ
Mathlib/Topology/Order.lean -> gi_generate_from / TopologicalSpace.gciGenerateFrom
Mathlib/Topology/Order/Basic.lean -> tendsto.eventually_lt / Filter.Tendsto.eventually_lt
Mathlib/Topology/UniformSpace/Basic.lean -> uniform_continuous_inf_rng / UniformContinuous.inf_rng
Mathlib/Topology/UniformSpace/Basic.lean -> uniform_continuous_inf_dom_left / UniformContinuous.inf_dom_left
Mathlib/Topology/UniformSpace/Basic.lean -> uniform_continuous_inf_dom_right / UniformContinuous.inf_dom_right
```

**Comment by eric-wieser** [comment]
> I needed to do it to get the full list of possible misalignments:

All the more reason to do this in a separate PR and get it merged first!

**Review by Ruben-VandeVelde — APPROVED** [review]

**Comment by ChrisHughes24** [comment]
bors merge

**Pull request closed without merging**